src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+	
+	private String value;
+    private boolean isPrivate;
+	
+	protected Contact(){
```

Comment:
think how you can actually use the constructor to set values by using a `super()` call from the child class?